src/microbe_stage/MicrobeStage.cs

```diff
@@ -190,6 +190,19 @@ public override void _Ready()
         SetupStage();
     }
 
+    public override void _Notification(int what)
+    {
+        if (what == NotificationTranslationChanged)
+        {
+            foreach (var patch in CurrentGame.GameWorld.Map.Patches)
```

Comment:
I think it would make sense to add a method in GameWorld called ApplyTranslations that would apply patch name translations, and also it could in the future apply all other translation changes that are needed.